### DISCUSSION TITLE
Status page empty in Minidlna 1.3.2

### DISCUSSION THREAD

**Discussion by donacarr** [Freetz-NG/freetz-ng#633] [Allgemein]
Hi there,

on 7590 with 7.29 upgrading freetz-ng from 18722-6e2a97c09 to 20254-342950c57  I noted that minidlna status is not showing anything...
Any idea?

![image](https://user-images.githubusercontent.com/13415391/198678069-eb8a333d-95db-4abc-9359-3e246ee5dee8.png)

**Comment by CaptainJackDaniels** [comment]
![Screenshot 2022-10-29 at 00-07-48 Freetz @7490-client – MiniDLNA](https://user-images.githubusercontent.com/111428641/198740674-306fead4-33d0-4586-a072-c583ab6a6e56.png)

**Comment by fda77** [comment]
@CaptainJackDaniels: Thx to show it works at all with current version.
@donacarr
https://github.com/Freetz-NG/freetz-ng/blob/master/make/pkgs/minidlna/files/root/usr/lib/cgi-bin/minidlna/minidlna_status.cgi#L24
The Status page is read here with wget. Maybe this is broken, your services does not run or is "unintended" configured

**Comment by donacarr** [comment]
I update to latest commit with no luck...
But I found this in the minidlna.log
` upnphttp.c:938: error: DNS rebinding attack suspected (Host: fritz.box:8200)`

minidlna is running on port 8200
```
netstat -an | grep 8200
tcp        0      0 0.0.0.0:8200            0.0.0.0:*               LISTEN
tcp        0      0 192.168.178.1:8200      192.168.178.20:49394    ESTABLISHED
```

Any idea?

**Comment by donacarr** [comment]
One note ....18722-6e2a97c09 was with minidlna 1.3.0 while now is 1.3.2.
Could be related to https://groups.google.com/g/linux.debian.bugs.dist/c/BZ0zzB_KAQE  ??
Does anyone has a working minidlna 1.3.2 ?

**Comment by fda77** [comment]
Oh yes, its security for the read-only status webif -.-
![image](https://user-images.githubusercontent.com/471850/200172865-17ad3c48-af4b-4214-a24c-1f764912f936.png)
Maybe it helps to use the IP with wget - even 127.0.0.1 

Can you try to replace `$(self_host)` by `127.0.0.1` here? https://github.com/Freetz-NG/freetz-ng/blob/master/make/pkgs/minidlna/files/root/usr/lib/cgi-bin/minidlna/minidlna_status.cgi#L24

You could test it on the device itself, by copy `/usr/lib/cgi-bin/minidlna/minidlna_status.cgi` to temp and mount it with `-o bind` writable

**Reply by fda77** [reply]
But im not sure if 127.0.0 still works (if it works) when an "interface" is configured in web interface

**Comment by fda77** [comment]
74d4e5bb5
